build.gradle.kts

```diff
@@ -1,11 +1,7 @@
-buildscript {
-    extra.apply {
-        set("compose_ui_version", "1.1.1")
-    }
-}// Top-level build file where you can add configuration options common to all sub-projects/modules.
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id("com.android.application") version "7.3.1" apply false
-    id("com.android.library") version "7.3.1" apply false
-    id("org.jetbrains.kotlin.android") version "1.6.10" apply false
+//    id("com.android.application") version "7.3.1" apply false
+//    id("com.android.library") version "7.3.1" apply false
+//    id("org.jetbrains.kotlin.android") version "1.6.10" apply false
```

Comment:
Is there a reason why these were commented out? Were they supposed to be deleted?

Reply:
Yeah they were supposed to be deleted. I commented them out initially to make sure the project compiled fine without them